app/main.py

```diff
@@ -24,6 +27,10 @@
     from typing import Any, Dict
 
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
 class AppSettings(RedisSettings):
     """Redis settings."""
 
```

Comment:
```suggestion

    AUTH_DEPENDENCIES: str = Field("", description="List of FastAPI dependencies for authentication features.")
```
You can even consider adding the parsing of this variable into this class as a `pydantic.validator` class method. But this is fine for now, for sure :)